openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -91,13 +91,25 @@ def _get_custom_pacing_children(subsection, num_weeks):
     Return relative date items for the subsection and its children
     """
     items = [subsection]
+    all_problems_are_ora = True
+    has_empty_unit = True
```

Comment:
This reads like a "negative" variable name to me, which can be a little more confusing than necessary. For example, `if not has_empty_unit` would take me a second to parse. Could it be restyled/flipped in a positive way, like `has_content`?